src/ontology/edits/oeo-physical.omn

```diff
@@ -9262,6 +9262,121 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1359",
         OEO_00140104
     
     
+Class: OEO_00310023
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A heat recovery steam generator is a heat exchanger that recovers heat from a hot steam.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "heat recovery steam generator"
+    
+    SubClassOf: 
+        OEO_00140102,
+        OEO_00000503 some OEO_00110001,
+        OEO_00010234 some OEO_00000007
+    
+    
+Class: OEO_00310025
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Combined cycle electricity generation is a fuel-powered electricity generation process in which a gas turbine process is followed by an steam-electric process.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "combined cycle electricity generation"
+    
+    SubClassOf: 
+        OEO_00050001,
+        OEO_00010234 some OEO_00000139,
+        OEO_00010234 some OEO_00240000,
+        <http://purl.obolibrary.org/obo/RO_0002427> some OEO_00050020,
+        <http://purl.obolibrary.org/obo/RO_0002427> some OEO_00310027
```

Comment:
> 'has energy input' some 'electrical energy'
> 'has energy input' some 'rotational energy'

No.  Chemical energy goes in, electrical energy (and possibly heat) come out.  Hence the name "… electricity generation".

Reply:
I will delete the both axioms 'has energy input' some 'electrical energy' and 'has energy input' some 'rotational energy'.
The axioms 'has energy input' some 'chemical energy' and 'has energy output' some 'electrical energy' are already added through the parent classes.